src/common/containers/build-details.js

```diff
@@ -38,12 +38,13 @@ class BuildDetails extends Component {
           snapcraft cleanbuild --debug
         </HelpInstallSnap>
       );
-    } else if (snap && snap.store_name) {
-      // otherwise if we have snap name show install instructions
+    } else if (snap && snap.store_name && build.storeRevision) {
```

Comment:
Right, in that case it's not possible to show install instructions.  (It might not be a failure - perhaps it's still in progress.)